src/lib/model/InboundTransfersModel.js

```diff
@@ -277,25 +315,35 @@ class InboundTransfersModel {
      * Validates an incoming transfer prepare request and makes a callback to the originator with
      * the result
      */
-    async prepareTransfer(prepareRequest, sourceFspId) {
+    async prepareTransfer(request, sourceFspId) {
+        const prepareRequest = request.body;
         try {
             // retrieve our quote data
-            let quote;
-
             if (this._allowDifferentTransferTransactionId) {
                 const transactionId = this._ilp.getTransactionObject(prepareRequest.ilpPacket).transactionId;
-                quote = await this._cache.get(`quote_${transactionId}`);
+                this.data = await this._cache.get(`transferModel_in_${transactionId}`);
             } else {
-                quote = await this._cache.get(`quote_${prepareRequest.transferId}`);
+                this.data = await this._cache.get(`transferModel_in_${prepareRequest.transferId}`);
             }
 
-            if(!quote) {
+            const quote = this.data.quote;
+
+            if(!this.data || !quote) {
+                // NOTE If you are using the adapter in as an old "mojaloop connector" it absolutely requires a
```

Comment:
Will do. I'll clarify this, edit my above PR comment and add a comment to the configuration file.

Basically mojaloop-connector removed a feature in the adapter since it was absolutely not needed. Since the merge you now need to explicitly have it turned off. I'll make that clear in a bit.